dali/python/backend_impl.cc

```diff
@@ -798,20 +818,25 @@ void ExposeTensor(py::module &m) {
 
   auto tensor_gpu_binding = py::class_<Tensor<GPUBackend>>(m, "TensorGPU")
     .def_property_readonly_static("__module__", tensor_module_impl)
-    .def(py::init([](py::capsule &capsule, string layout = "") {
+    .def(py::init([](py::capsule &capsule, string layout, py::object stream) {
           auto t = std::make_unique<Tensor<GPUBackend>>();
           FillTensorFromDlPack(capsule, t.get(), layout);
+          if (!stream.is_none())  // use a separately provided stream - there's none in the capsule
+            t->set_order(AccessOrderFromPythonStreamObj(stream));
```

Comment:
What's the order of  `*t` at the L824? Does calling `set_order` in L825 without `synchronize=False` add unnecessary synchronization of the `stream` with that default order?

Reply:
It's CPU, so there's no synchronization.